ntbs-service/Pages/LabResults/Index.cshtml.cs

```diff
@@ -114,20 +114,33 @@ public async Task<IActionResult> OnGetPotentialMatchDataAsync(string manualNotif
             if (!ValidationService.IsValid(combinedModelPropertyKey))
             {
                 var errorMessage = ModelState[combinedModelPropertyKey].Errors.First().ErrorMessage;
-                return new JsonResult(new { errorMessage });
+                return new JsonResult(new { errorMessage, notificationIdSpecified = true });
+            }
+
+            if (string.IsNullOrWhiteSpace(manualNotificationId))
+            {
+                return new JsonResult(new { notificationIdSpecified = false });
             }
 
             var parsedManualNotificationId = int.Parse(manualNotificationId);
```

Comment:
So this line here was the original problem, the validation above allows for empty or whitespace strings (mainly because it is maintained at all times, not just when the row's radio button is selected), and when you are selecting an existing match you don't need to fill the field in.

My approach was to add a flag to return result explaining that no notification ID was provided, not as an error (because we would turn the box red as soon as we loaded the page), but as a separate flag to indicate that the previous notification preview, and any ID validation messages, can both be removed, without any new things needing to be displayed.